src/tools/plan-generator.ts

```diff
@@ -116,11 +117,16 @@ export class PlanGenerator {
       return ktUtils.applyFun('StorageKeyParser.parse', [quote(handle.storageKey.toString())]);
     }
     if (handle.fate === 'create') {
-      return ktUtils.applyFun('CreateableStorageKey', [quote(handle.id)]);
+      return ktUtils.applyFun('CreateableStorageKey', [quote(handle.id || this.createHandleName())]);
     }
     throw new PlanGeneratorError(`Problematic handle '${handle.id}': Only 'create' Handles can have null 'StorageKey's.`);
   }
 
+  createHandleName(): string {
+    const rand = Math.floor(Random.next() * Math.pow(2, 50));
```

Comment:
since these handles aren't mappable into other arcs, the IDs only need to be unique within the given recipe, right? so in theory you could just enumerate them - `handle0`, `handle1` etc.
when the Allocator creates these storage keys, they are tied to the unique arc's ID anyway (@cromwellian  to confirm)